### ISSUE TITLE
Exporting Graphics to React on unix fails because of case sensitive file names

### ISSUE THREAD

**Issue by mcastiello** [mikaelvesavuori/figmagic#114]
I was trying to set up a GitHub pipeline following [this example](https://github.com/mikaelvesavuori/github-ci-demo) and using the [default template](https://www.figma.com/community/file/821094451476848226).

this is my currect `figmagic.json`
```json
{
  "outputFormatTokens": "json",
  "outputFormatStorybook": "mdx",
  "outputFolderElements": "./src/elements",
  "outputFolderGraphics": "./src/graphics",
  "outputFolderTokens": "./src/tokens",
  "syncElements": true,
  "syncGraphics": true,
  "syncTokens": true,
  "refreshType": "hard",
  "templates": {
    "templatePathReact": "./templates/react",
    "templatePathGraphic": "./templates/graphic",
    "templatePathStyled": "./templates/styled"
  },
  "tokensRelativeImportPrefix": "../../../",
  "outputGraphicElements": true,
  "overwrite": {
    "css": true,
    "description": true,
    "graphic": true,
    "react": true,
    "storybook": true,
    "styled": true
  }
}
```

This is the output I get from GitHub:

![image](https://user-images.githubusercontent.com/12670283/135251267-9cd68248-86b0-4b83-bdbd-0ab323714bd4.png)

This process works flawless on my local windows machine, so I've added a few extra steps to see if the Graphics where generated:

![image](https://user-images.githubusercontent.com/12670283/135251605-8ae89f63-e772-4f0d-8552-c7219b612448.png)

I've started debugging the code and I've noticed that this is the piece of code that it is throwing the error in `bin/usecases/interactors/elements/writeElements.ts`:

```ts
      if (type === 'graphic') {
        const SVG_DATA = getSvgFileData(`${config.outputFolderGraphics}/${config.name}.svg`);
        return cleanSvgData(SVG_DATA);
      }
```

And `config.name` is defined like this:
```ts
const name = toPascalCase(element.name);
```

so, basically, it is looking for `./src/graphics/Check.svg` instead of `./src/graphics/check.svg`.\
This works fine on a windows machine, but it won't work on unix.

**Desktop:**
 - OS: ubuntu-latest
 - Node: 14.x
 - Version: 4.3.3

I guess exporting the actual SVG using Pascal Case notation would potentially fix the issue.

**Comment by mikaelvesavuori** [comment]
Hi! I will look into this after the other ongoing issues. This issue shouldn't happen and was supposed to be covered by other recent work, though.... :/

**Comment by mcastiello** [comment]
Thanks, your work is very much appreciated.
For now I just wrote a temporary script that reads the graphics path and generate the react components.

**Comment by mcastiello** [comment]
Lol, sorry, I hit the wrong button and closed the issue accidentally

**Comment by mikaelvesavuori** [comment]
I was starting work on this last night and it is related to one other current issue.

One way to deal with this is by simply not using transforms like toPascalCase() on anything. Probably that will be my ”solution”, thus exactly representing whatever names one might use in Figma.

**Comment by mikaelvesavuori** [comment]
I am trying to push out 4.3.5 now which should fix this.

This release changes the previous behaviour of using pascal case or camelcase to adjust names that are passed through. The new behaviour is that names are taken "as-is".

The change should fix your exhibited issue.
